Gemfile

```diff
@@ -31,7 +31,6 @@ gem 'figaro'
 gem 'graphql'
 gem 'search_object_graphql'
 gem 'skylight'
-gem 'sprockets'
```

Comment:
Let's look into this tomorrow and see why we may or may not need this gem!

Reply:
This seemed to be giving me trouble when trying to create the database. If we need to add it back in later that is fine.